dashboard/app/controllers/scripts_controller.rb

```diff
@@ -13,6 +13,9 @@ class ScriptsController < ApplicationController
   use_reader_connection_for_route(:show)
 
   def show
+    if @script.is_deprecated
```

Comment:
Can you also add this to the edit action? I worry that if someone edited the unit for some reason it might reverse the `is_deprecated` field (not sure but this unit shouldn't be edited anyway)

Reply:
done!